dask_cuda/__init__.py

```diff
@@ -9,6 +31,33 @@
 import dask.dataframe.shuffle
 import dask.dataframe.multi
 import dask.bag.core
+import os
+
+
+def is_amd_gpu_available():
+    """Utility method to check if AMD GPU is available"""
+    import subprocess
+    import re
+
+    check_cmd ='rocminfo'
+    try:
+        proc_complete = subprocess.run(
+            check_cmd.split(),
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            check=True,
+        )
+        for line in proc_complete.stdout.decode('utf-8').split():
+            if re.search(r"(gfx908|gfx90a|gfx940|gfx941|gfx942|gfx1100)", line):
+                return True
+        return False
+    except (FileNotFoundError, subprocess.CalledProcessError) as err:
+        print('  Error => ', str(err))
+        return False
+
+
+DASK_USE_ROCM = is_amd_gpu_available()
```

Comment:
@younseojava  IMO you could assume that an AMD GPU is available when someone uses this code.